src/Microsoft.Sbom.Api/Executors/ComponentDetectionBaseWalker.cs

```diff
@@ -35,8 +35,10 @@ public abstract class ComponentDetectionBaseWalker
     private readonly ISbomConfigProvider sbomConfigs;
     private readonly IFileSystemUtils fileSystemUtils;
     private readonly ILicenseInformationFetcher licenseInformationFetcher;
+    private readonly IPackageDetailsFactory packageDetailsFactory;
 
     public ConcurrentDictionary<string, string> LicenseDictionary = new ConcurrentDictionary<string, string>();
+    public ConcurrentDictionary<(string, string), PackageDetailsObject> PackageDetailsDictionary = new ConcurrentDictionary<(string, string), PackageDetailsObject>();
```

Comment:
nit: shorter syntax should be available
```csharp
public ConcurrentDictionary<(string, string), PackageDetailsObject> PackageDetailsDictionary = new ();
```

Reply:
Actually, do you even need this object to be publicly accessed property? I only see usage in `Scan()` method, so maybe it can simply be variable defined in this method?